### ISSUE TITLE
API Documentation

### ISSUE THREAD

**Issue by bunnyhero** [superpowers/docs.superpowers-html5.com#13]
Please add API documentation for at least the core game system! The justification for the lack of API documentation ("Since Superpowers is made out of many plugins and you can customize them and install your own, we don't provide an online API documentation") is simply unacceptable. Browsing the TypeScript API lets you find classes and methods, but there is no description for 99.9% of them.

**Comment by elisee** [comment]
Hi!

Thanks for the report. The TypeScript API browser should be improved, I created an issue https://github.com/superpowers/superpowers-game/issues/137 to track that.

Once that is done, someone can feel free to setup & keep an online API website up-to-date, but the real problem is that our built-in API browser is too basic and we don't use any in-code documentation convention right now.

Closing in favor of https://github.com/superpowers/superpowers-game/issues/137, feel free to discuss there!
